### PR TITLE
Reduse loading time for schedule mission dialog

### PR DESCRIPTION
**Pull request by Eddasol**

### CODE REVIEW

**Review by Christdej — CHANGES_REQUESTED** [review]

File: backend/api/Controllers/Models/EchoMissionOption.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿# nullable disable
+namespace Api.Controllers.Models
+{
+    public class EchoMissionOption
```
[the remark this answers is not in the document]
Comment (Eddasol) [reply backend/api/Controllers/Models/EchoMissionOption.cs:4]: Same, I'm not a fan of the name. 

Its basicly the same as the missionTemplate/missionDefinition that has been discussed. Maybe we should name it that and create a card to add the source functionality?

[the remark this answers is not in the document]
Comment (Christdej) [reply backend/api/Controllers/Models/EchoMissionOption.cs:4]: Yeah, that sounds like a good idea to me =)

[the remark this answers is not in the document]
Comment (Eddasol) [reply backend/api/Controllers/Models/EchoMissionOption.cs]: Changed it to MissionDefinition with echoMissionId as a "placeholder" for source

**Review by Christdej — APPROVED** [review]
LGTM =)

**Pull request merged**